installer/roles/create-mobile-client-crd/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+# tasks file for create-mobile-client-crd
+
+- name: Copy CRD definition to tmp file
+  template:
+    src: ../files/mobile-client-crd.yaml
+    dest: /tmp/mobile-client-crd.yaml  
+
+- name: Create mobileClient CRD
+  shell: "oc create -f /tmp/mobile-client-crd.yaml"
```

Comment:
Will this result in an AlreadyExists error if the playbook is run twice in a row without tearing down the cluster inbetween? Maybe we could use `failed_when` and check for AlreadyExists if it does.

Reply:
Good point, I just haven't had the need to re-run the playbook over an already existing cluster so it wasn't on my radar